src/main/java/net/pms/network/HTTPResource.java

```diff
@@ -44,6 +44,7 @@
 	public static final String UNKNOWN_AUDIO_TYPEMIME = "audio/mpeg";
 	public static final String AUDIO_TRANSCODE = "audio/transcode";
 	public static final String VIDEO_TRANSCODE = "video/transcode";
+	public static final String AUDIO_AAC_TYPEMIME = "audio/mp4";
```

Comment:
This is wrong, the mime type for AAC is ```audio/aac``` (it only applies to raw AAC streams with the ```.aac``` extension).